tests/integration/models/device-type.spec.ts

```diff
@@ -4,11 +4,14 @@ import { balena, givenLoggedInUser } from '../setup';
 import { timeSuite } from '../../util';
 import type * as BalenaSdk from '../../..';
 
-const DEVICE_TYPE_NAME = 'Raspberry Pi 2';
-const DEVICE_TYPE_SLUG = 'raspberry-pi2';
-const DEVICE_TYPE_ALIAS = 'raspberrypi2';
-const DEVICE_TYPE_INSTALL_METHOD = 'externalBoot';
-const DEVICE_TYPE_ID = 1;
+const RPI2_DEVICE_TYPE_NAME = 'Raspberry Pi 2';
+const RPI2_DEVICE_TYPE_SLUG = 'raspberry-pi2';
+const RPI2_DEVICE_TYPE_ALIAS = 'raspberrypi2';
+const RPI2_DEVICE_TYPE_INSTALL_METHOD = 'externalBoot';
+const RPI2_DEVICE_TYPE_ID = 1;
+
+const RADXA_ZERO_DEVICE_TYPE_SLUG = 'radxa-zero';
+const RADXA_ZERO_DEVICE_TYPE_INSTALL_METHOD = 'radxaFlash';
```

Comment:
@acostach @vipulgupta2048 given v
See: https://github.com/balena-io/contracts/pull/366
and that the tests atm fail w/ a related complain, should you change the expected installation method to be `usbMassStorage`?


>       balena.models.deviceType.getInstallMethod()
>         should get device type installation method:
>
>      AssertionError: expected 'usbMassStorage' to equal 'radxaFlash'
>      + expected - actual
>
>      -usbMassStorage
>      +radxaFlash

Reply:
Feel free to self-certify once you fix the tests :+1:

Reply:
I see the issue, fixing